nullaway/src/main/java/com/uber/nullaway/ASTHelpersBackports.java

```diff
@@ -0,0 +1,38 @@
+package com.uber.nullaway;
+
+import com.sun.tools.javac.code.Symbol;
+import java.util.List;
+
+/**
+ * Methods backported from {@link com.google.errorprone.util.ASTHelpers} since we do not yet require
+ * a recent-enough Error Prone version. The methods should be removed once we bump our minimum Error
+ * Prone version accordingly.
+ */
+public class ASTHelpersBackports {
+
+  private ASTHelpersBackports() {}
+
+  /**
+   * Returns true if the symbol is static. Returns {@code false} for module symbols. Remove once we
+   * require Error Prone 2.16.0 or higher.
+   */
+  @SuppressWarnings("ASTHelpersSuggestions")
+  public static boolean isStatic(Symbol symbol) {
+    if (symbol.getKind().name().equals("MODULE")) {
+      return false;
+    }
+    return symbol.isStatic();
+  }
+
+  /**
+   * A wrapper for {@link Symbol#getEnclosedElements} to avoid binary compatibility issues for
+   * covariant overrides in subtypes of {@link Symbol}.
+   *
+   * <p>Same as this ASTHelpers method in Error Prone:
+   * https://github.com/google/error-prone/blame/a1318e4b0da4347dff7508108835d77c470a7198/check_api/src/main/java/com/google/errorprone/util/ASTHelpers.java#L1148
+   * TODO: delete this method and switch to ASTHelpers once we can require Error Prone 2.20.0
+   */
```

Comment:
See [here](https://github.com/google/error-prone/issues/2092#issuecomment-759096230).  Sometime after JDK 11, some subclasses of `Symbol` were changed to override `getEnclosedElements()` with a covariant return type (returning `com.sun.tools.javac.util.List` instead of `java.util.List`).  So, e.g., if your code directly invokes `getEnclosedElements()` on a variable of type `ClassSymbol`, and you compile on JDK 17, the bytecode signature at the call site will have a return type of `com.sun.tools.javac.util.List`.  Then when you try to run this code on JDK 11, it fails, since a method with that exact return type cannot be found.  This issue occurs even if you set your source and target bytecode level to 11 (or 8).  Using this wrapper method works around the problem, since the declared target of the call will always be `Symbol.getEnclosedElements()`.